bundles/org.openhab.binding.saicismart/src/main/java/org/openhab/binding/saicismart/internal/asn1/Util.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.saicismart.internal.asn1;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ *
+ * @author Markus Heberling - Initial contribution
+ */
+@NonNullByDefault
+public final class Util {
+    private Util() {
+    }
+
+    public static void fillReserved(byte[] reservedBytes) {
```

Comment:
Please add some javadoc

Reply:
I investigated this class and there is a util in the saic-ismart-api lib that has a very similar method.  This method is called on the initializeMessage so the "ReservedBytes" are filled and the Util class is not needed.  I tested without this fillReserved and the binding works. So I removed rather than documented the class.